h/models/group_scope.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import sqlalchemy as sa
+
+from h.db import Base
+
+
+class GroupScope(Base):
+    __tablename__ = 'groupscope'
```

Comment:
Would be useful to have a docstring here explaining what a `GroupScope` is for the benefit of newcomers to the codebase.

Reply:
Good idea added